tensorbay/label/attributes.py

```diff
@@ -69,6 +69,9 @@ class Items(ReprMixin, EqMixin):
     When the value type of an attribute is array,
```

Comment:
What's the meaning of backquotes？

Reply:
docs(label): use inline code format for arguments in attributes.py
docs(label): add "TODO" directive for the incorrect display of "type_"